src/dotnet/Mgmt.CI.BuildTools/ci.yml

```diff
@@ -26,11 +26,9 @@ stages:
           vmImage: 'windows-2022'
 
         steps:
-          - task: UseDotNet@2
-            displayName: 'Use .NET Core sdk $(DotNetCoreVersion)'
-            inputs:
-              packageType: sdk
-              version: $(DotNetCoreVersion)
+          - template: /eng/pipelines/templates/steps/install-dotnet.yml
+            parameters:
+              DotNetCoreVersion: $(DotNetCoreVersion)
```

Comment:
We should let this use the default version and we should even consider removing DotNetCoreVersion from globals.yml

Reply:
Got it. I will investigate removal of `DotNetCoreVersion` from `globals.yml` in a separate PR.

Reply:
@weshaggard I went ahead and [removed it in this PR](https://github.com/Azure/azure-sdk-tools/pull/5045/commits/baa32b74a6d1ec2211e5809bc59c5e4603d11a7d).